<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseDueDate</class>
 <widget class="QWidget" name="ChooseDueDate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>466</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>315</width>
    <height>466</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>315</width>
    <height>466</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QPushButton" name="removeDueDate">
      <property name="text">
       <string>移除到期</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="chooseDueDateSureButton">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCalendarWidget" name="chooseDueDateEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>296</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
